### PR TITLE
exportable expense API and date range filter for expense group view  API

### PR DESCRIPTION
**Pull request by Ashutosh619-sudo**

### CODE REVIEW

File: apps/fyle/views.py [hunk 1]
```diff
@@ -53,6 +77,31 @@ def get_queryset(self):
...
+        if configuration.reimbursable_expenses_object:
+            fund_source.append('PERSONAL')
+        if configuration.corporate_credit_card_expenses_object:
+            fund_source.append('CCC')
+
+        expense_group_ids = ExpenseGroup.objects.filter(
+            workspace_id=self.kwargs['workspace_id'],
+            exported_at__isnull=True,
+            fund_source__in=fund_source
+        ).values_list('id', flat=True)
```
Comment (NileshPant1999) [inline apps/fyle/views.py:97]: can move all of the above code to a helper function and get ids from there
can help make the view look a bit cleaner

File: tests/test_fyle/test_views.py [hunk 2]
```diff
@@ -382,3 +389,16 @@ def test_expenses(mocker, api_client, access_token):
...
+def test_exportable_expense_group_view(api_client, access_token):
+
+   url = '/api/workspaces/1/fyle/exportable_expense_groups/'
+   api_client.credentials(HTTP_AUTHORIZATION='Bearer {}'.format(access_token))
+
+   response = api_client.get(url)
+   assert response.status_code==200
+
+   response = json.loads(response.content)
+   assert response['exportable_expense_group_ids'] == [1, 2]
```
Comment (NileshPant1999) [inline tests/test_fyle/test_views.py:404]: new line

File: apps/fyle/actions.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+from apps.fyle.models import ExpenseGroup
+from apps.workspaces.models import Configuration
+
+
+def get_expense_group_ids(workspace_id, ):
```
[the remark this answers is not in the document]
Comment (ashwin1111) [reply apps/fyle/actions.py:5]: remove extra comma in arg

[the remark this answers is not in the document]
Comment (ashwin1111) [reply apps/fyle/actions.py:5]: name it get_exportable_expense_group_ids()

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**